main/api.c

```diff
@@ -19,6 +19,22 @@
 #include <sys/un.h>
 #include <unistd.h>
 
+static evutil_socket_t *notification_sockets;
+
+void gr_api_push_notification(struct gr_api_notification *n) {
+	size_t len = sizeof(n) + n->payload_len;
+	struct gr_api_response r = {
+		.for_id = -1,
+		.status = 0,
+		.payload_len = len,
+	};
+	evutil_socket_t s;
+	gr_vec_foreach (s, notification_sockets) {
+		send(s, &r, sizeof(r), MSG_DONTWAIT | MSG_NOSIGNAL);
+		send(s, n, len, MSG_DONTWAIT | MSG_NOSIGNAL);
```

Comment:
Does `MSG_NOSIGNAL` mean that we don't detect when clients disconnect while we are sending notifications? Or is it detected in `read_cb`?